README.md

```diff
@@ -12,9 +12,16 @@ holds a list of 128-bit GATT UUIDs that
 websites using the Web Bluetooth API are forbidden from accessing.
 This includes all of Services, Characteristics, and Descriptors.
 
-This file contains comments in lines starting with `#`.
-Non-comment lines contain a [valid UUID](https://webbluetoothcg.github.io/web-bluetooth/#valid-uuid)
+[manufacturer_data_blocklist.txt](/manufacturer_data_blocklist.txt)
+holds a list of manufacturer data that websites using the Web Bluetooth API are forbidden from accessing.
+Each entry of the list is consist of company identifier and data prefix that should be blocked.
+
+These files contain comments in lines starting with `#`.
+Non-comment lines contain one of these:
+  * A [valid UUID](https://webbluetoothcg.github.io/web-bluetooth/#valid-uuid)
 followed optionally by a space and one of the tokens "`exclude-reads`" or "`exclude-writes`".
+  *  The word `manufacturer` followed by a space and a group of hexadecimal digits (at most four hex digits)
+  that represents a company identifier. A space after that, is `advdata-` followed by a data prefix with mask.
```

Comment:
The advertising data syntax would benefit from more description.  Say in words "Advertising data" so people don't have to guess or parse advdata, and describe the way the data and mask are written?

Reply:
Added more detail to explain the data prefix format and how to match an advertising data.